spec/turn_spec.rb

```diff
@@ -0,0 +1,258 @@
+        card5 = Card.new(:heart, '8', 8)   
+        card6 = Card.new(:diamond, 'Queen', 12) 
+        card7 = Card.new(:heart, '3', 3)
+        card8 = Card.new(:diamond, '2', 2)
+
+        deck1 = Deck.new([card1, card2, card5, card8])
+        deck2 = Deck.new([card3, card4, card6, card7])
+
+        player1 = Player.new("Megan", deck1)
+        player2 = Player.new("Aurora", deck2)
+
+        turn = Turn.new(player1, player2)
+                
+        expect(turn.type).to be(:basic)
+    end
+
+    it 'has a winner' do
+        card1 = Card.new(:heart, 'Jack', 11)
+        card2 = Card.new(:heart, '10', 10)
+        card3 = Card.new(:heart, '9', 9)
+        card4 = Card.new(:diamond, 'Jack', 11)
+        card5 = Card.new(:heart, '8', 8)   
+        card6 = Card.new(:diamond, 'Queen', 12) 
+        card7 = Card.new(:heart, '3', 3)
+        card8 = Card.new(:diamond, '2', 2)
+
+        deck1 = Deck.new([card1, card2, card5, card8])
+        deck2 = Deck.new([card3, card4, card6, card7])
+
+        player1 = Player.new("Megan", deck1)
+        player2 = Player.new("Aurora", deck2)
+
+        turn = Turn.new(player1, player2)
+        turn.type           
+
+        expect(turn.winner).to be(player1)
+    end
+
+    it 'adds cards to spoils of war' do
+        card1 = Card.new(:heart, 'Jack', 11)
+        card2 = Card.new(:heart, '10', 10)
+        card3 = Card.new(:heart, '9', 9)
+        card4 = Card.new(:diamond, 'Jack', 11)
+        card5 = Card.new(:heart, '8', 8)   
+        card6 = Card.new(:diamond, 'Queen', 12) 
+        card7 = Card.new(:heart, '3', 3)
+        card8 = Card.new(:diamond, '2', 2)
+
+        deck1 = Deck.new([card1, card2, card5, card8])
+        deck2 = Deck.new([card3, card4, card6, card7])
+
+        player1 = Player.new("Megan", deck1)
+        player2 = Player.new("Aurora", deck2)
+
+        turn = Turn.new(player1, player2)
+        turn.type 
+        turn.pile_cards
+        expect(turn.spoils_of_war).to eq([card1, card3])
+    end
+
+    it 'awards spoils of war to winners deck' do
+        card1 = Card.new(:heart, 'Jack', 11)
+        card2 = Card.new(:heart, '10', 10)
+        card3 = Card.new(:heart, '9', 9)
+        card4 = Card.new(:diamond, 'Jack', 11)
+        card5 = Card.new(:heart, '8', 8)   
+        card6 = Card.new(:diamond, 'Queen', 12) 
+        card7 = Card.new(:heart, '3', 3)
+        card8 = Card.new(:diamond, '2', 2)
+
+        deck1 = Deck.new([card1, card2, card5, card8])
+        deck2 = Deck.new([card3, card4, card6, card7])
+
+        player1 = Player.new("Megan", deck1)
+        player2 = Player.new("Aurora", deck2)
+
+        turn = Turn.new(player1, player2)
+        turn.type 
+        winner = turn.winner
+        turn.pile_cards
+        turn.award_spoils_of_war(winner)
+        expect((player1.deck.cards - [card2, card5, card8, card1, card3]).empty?).to eq(true)
+        expect((player2.deck.cards - [card4, card6, card7]).empty?).to eq(true)
```

Comment:
I do not understand the expectation on lines 161 and 162. If `player1` wins then they should get all the cards in the spoils, so how is their deck empty? same with `player2`.